R/sidebar.R

```diff
@@ -199,6 +218,14 @@ sidebar_close <- function(id, session = get_current_session()) {
   session$onFlush(callback, once = TRUE)
 }
 
+collapse_icon <- function() {
+  if (!rlang::is_installed("bsicons")) {
```

Comment:
Use internal `is_available()`

```suggestion
  if (!is_available("bsicons")) {
```